oshi-core/src/main/java/oshi/software/os/OSProcess.java

```diff
@@ -362,6 +364,12 @@
      */
     boolean updateAttributes();
 
+    /**
+     * Retrieves the threads of the process and their details.
+     * @return a list of threads
```

Comment:
Let's get a little verbose here. Copy one of the others, should probably be "returns an unmodifiable list of OSThreads" or similar.

Reply:
Will take it as a general review for all the java docs and apply where ever possible.